src/ngx_pagespeed.cc

```diff
@@ -3019,13 +3019,17 @@ ngx_int_t ps_preaccess_handler(ngx_http_request_t* r) {
   ph = cmcf->phase_engine.handlers;
 
   i = r->phase_handler;
+
   // move handlers before try_files && content phase
+  // As of nginx 1.13.4 we will be right before the try_files module
+  #if (nginx_version < 1013004)
```

Comment:
@jmarantz I was a bit too eager with merging this, opened https://github.com/pagespeed/ngx_pagespeed/pull/1458 for addressing this